.wercker/condition-wercker.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (pluginConfig, config, cb) {
+  const env = config.env;
+
+  if (env.WERCKER !== 'true') {
```

Comment:
there is no env var `WERCKER`, there is a `CI` var (see https://devcenter.wercker.com/administration/environment-variables/available-env-vars/)

Reply:
The script is based on this: https://github.com/io-monad/sr-condition-wercker/blob/master/index.js

They also have a similar condition.

Reply:
Maybe we should have the script checking the two env vars?

Reply:
I have also find that `CI` variable is set to true only in the `build` pipeline.
https://app.wercker.com/APIPCS/blueprint-markdown-renderer/runs/build/5b744ae8f3e12d00071b7c2a?step=5b744ae8f3e12d00071b7c33
In the next workflow it is not available - if it is not set in environment variables:
https://app.wercker.com/APIPCS/blueprint-markdown-renderer/runs/build-v6/5b744afe74452b00062c4f90?step=5b744afe74452b00062c4f92

The variable `WERCKER` is set in all pipelines.

Reply:
Interesting. We should report that to them, explaining the use-case. They might change it.

Reply:
@patricksmms if you check the documentation Nada mentioned
https://devcenter.wercker.com/administration/environment-variables/available-env-vars/
There are 2 parts:
1. `Git-type Workflow related variables` - variable `CI` is present
2. `Workflow-type related variables` - variable `CI` is not present, `variable `DEPLOY` is set.

if you check those examples from my last comment. You can see that variables in first pipeline `build` (run by github hooks) are setup as in doc part 1. For all other pipelines variables are set according to part 2 in doc.

Reply:
Well, this step (on this app) isn't in a `build` pipeline, so you can't check for the `CI` env var. In other projects it might be (currently is on the `ui-kit`). The `wercker` var is always present, although not documented. I would leave it as it is, but its your call ;)